go.mod

```diff
@@ -264,6 +264,7 @@ replace (
 	github.com/go-openapi/swag => github.com/go-openapi/swag v0.19.5
 	github.com/go-openapi/validate => github.com/go-openapi/validate v0.19.5
 	github.com/go-ozzo/ozzo-validation => github.com/go-ozzo/ozzo-validation v3.5.0+incompatible
+	github.com/go-sql-driver/mysql => github.com/go-sql-driver/mysql v1.5.0
```

Comment:
why are we adding a mysql driver to our dependency tree?

Reply:
looks like this was added in https://github.com/aws/aws-sdk-go/pull/3118/files#diff-37aff102a57d3d7b797f152915a6dc16R4 in v1.29.0+

this is a good example of the types of dependencies we don't want to include in the kubernetes/kubernetes dependency tree

Reply:
It looks like this is only used in an example file inside the aws repo here:

https://github.com/aws/aws-sdk-go/blob/master/example/service/rds/rdsutils/authentication/iam_authentication.go#L11

The AWS library should isolate that example to prevent all downstream consumers of the AWS sdk from inheriting a mysql driver dependency. Possible ways to do that:

* rename the authentication examples folder to `_authentication`, then run `go mod tidy` on the root (prevents inheriting mysql dependency, unblocking this PR)
* rename the example folder to to `_example`, then run `go mod tidy` on the root (prevents inheriting any example dependencies, which seems ideal)
* create a go.mod file inside the authentication or examples folder to isolate that into a separate module

Reply:
Is there any way around this, or the best that could be done is to use the latest v1.28.x?

Reply:
The AWS library maintainers have to fix this, reported in https://github.com/aws/aws-sdk-go/issues/3548

Reply:
> Is there any way around this, or the best that could be done is to use the latest v1.28.x?

it looks like v1.28.14 is the latest that can be used that does not have this issue

Reply:
Thanks. Will wait and see what the aws-sdk-go maintainers say.